jung-algorithms/src/main/java/edu/uci/ics/jung/layout/util/RadiusNetworkNodeAccessor.java

```diff
@@ -81,19 +68,18 @@ public N getNode(LayoutModel<N, P> layoutModel, P p) {
    * @return a node which is associated with the location {@code (x,y)} as given by {@code layout}
    */
   @Override
-  public N getNode(LayoutModel<N, P> layoutModel, double x, double y, double z) {
+  public N getNode(LayoutModel<N> layoutModel, double x, double y) {
     //    long time = System.currentTimeMillis();
     double minDistance = maxDistance * maxDistance * maxDistance;
```

Comment:
I need to revisit the original code

Reply:
I suspect that that's been around for a long time, probably since that class's ancestor was created; I wouldn't worry about going back in time on that.  I'd be fine with just going ahead and making that change; semantically what we want is in fact something that's effectively infinite.